packages/contracts-bedrock/justfile

```diff
@@ -62,8 +62,8 @@ test-dev *ARGS: build-go-ffi
 
 # Default block number for the forked upgrade path.
 
```

Comment:
Per a comment on ForkLive, I'm not exactly sure how to set these numbers, should it be after or before the most recent upgrade?
Would be good to clarify in a comment here.

Reply:
It should really be after the most recent upgrade, but it's before because Base isn't upgraded yet.